### ISSUE TITLE
Why chaosblade pod is sidecar?

### ISSUE THREAD

**Issue by NigelWu95** [chaosblade-io/chaosblade-operator#112]
## Issue Description

Type: *bug report* or *feature request*

### Describe what happened (or what feature you want)
chaosblade-tool is daemonset, chaosblade-operator is operator, when we create a experiment that is a blade cr not a container.
Sidecar is a side container with business/service container in the same pod. I think there is no sidecar role, could I sincerely get an explanation for this？
![image](https://user-images.githubusercontent.com/20470818/142381216-a28ea370-0b41-4421-b034-912ea6560ef8.png)

### How to reproduce it (as minimally and precisely as possible)

1. 
2. 
3.

**Comment by xcaspar** [comment]
In order to solve the problem that the tc command does not exist in the chaos target container. This Pod is not necessary.

**Comment by NigelWu95** [comment]
> In order to solve the problem that the tc command does not exist in the chaos target container. This Pod is not necessary.

To solve problem about tc command, is not depends on chaosblade-tool？But it is a daemonset😂
![image](https://user-images.githubusercontent.com/20470818/142559794-0a485850-dc9f-4eaf-8a87-4a068bc3c9d7.png)

**Comment by xcaspar** [comment]
You can set `daemonset.enable=false` to disable chaosblade-tool deployment.

**Comment by NigelWu95** [comment]
> You can set `daemonset.enable=false` to disable chaosblade-tool deployment.

Yeah, I got this, but chaosblade-tool is daomonset, there is no sidecar in the architecture?😂

**Comment by NigelWu95** [comment]
> You can set `daemonset.enable=false` to disable chaosblade-tool deployment.

Sorry, after I read the chaosblade-operator and chaosblade-exec-docker source code, I found the sidecar, thank you.
